src/index.ts

```diff
@@ -236,10 +236,14 @@ export function getBuildThroughStream(
 
 	// Regex to match against the type of command, e.g. FROM, RUN, COPY
 	const stepCommandRegex = /^\s?(\w+)(:?\s)/i;
-	// Use type-coercion here to suppress TS `may be undefined` warnings, as we know
-	// that function is only called with a value that will produce a non-undefined value
```

Comment:
I love that this comment is literally explaining how we'll never need this fix

Reply:
:flushed: Yeah... I wish I knew what did match with the first check, and not the second - I was so sure :p